src/firmware/boards/robot_stm32h7/io/robot_status.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+/**
+ * The robot status task is responsible for querying all the sensors
+ * and peripherals for their most up-to-date readings and packaging
+ * them in a RobotStatus proto. It then notifies the networking
+ * task to send out the updated proto.
+ *
+ * @param argument The robot status sender communication profile
```

Comment:
Given this is a void pointer, we might want to go into more detail on how this should be setup?

Reply:
added a comment